app/validators/multiparam_date_validator.rb

```diff
@@ -0,0 +1,73 @@
+class MultiparamDateValidator < ActiveModel::EachValidator
+  DATE_STRUCT = Struct.new('DateStruct', :day, :month, :year, keyword_init: true)
+
+  DEFAULT_OPTIONS = {
+    earliest_year: 1920,
```

Comment:
Is there anyone over 102 that might want legal aid? Pretty unlikely but you never know? perhaps 1900 might be better?

Reply:
Haha! I considered between the MVP development time and the moment we go live with a broader group of providers (not just closed beta with a handful of them) that hypothetical person might be 104 years old by then.

I can change to 1900 to be safe, 122 years old people have all the right to apply 😄

Reply:
They do indeed 👍